Annotation/polar_golden_set.json

```diff
@@ -402,7 +427,19 @@
                     }
                 ],
                 "type": "polar",
-                "clause": "",
+                "clause": "whether they innovate",
+                "embedded clauses": []
+            },
+            {
+                "predicate": [
+                    {
+                        "str": "ask",
+                        "lemma": "",
+                        "POS": ""
+                    }
+                ],
+                "type": "constituent",
+                "clause": "in what forms",

```

Comment:
I tend to agree that this is a constituent interrogative embedded clause, but it's a tricky case

Reply:
Yep, will add to "questions to ask" section in the monster doc